### PR TITLE
[asana] Revert Version

### PR DESCRIPTION
**Pull request by shmuel44**
## Description

Revert the `asana` library version from 5.07 to 3.2.1 due to backward compatibility issues affecting the `AsanaConnect` integration when using the native image.
[Asana forum](https://forum.asana.com/t/module-asana-has-no-attribute-client/496534)

image

### CODE REVIEW

**Review by samuelFain — APPROVED** [review]
LGTM

**Review by JudahSchwartz — CHANGES_REQUESTED** [review]
lts check that we cant fix the content first

**Comment by shmuel44** [comment]
> lts check that we cant fix the content first

the pack support level is `community`, so....

File: docker/py3-tools/pyproject.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ exifread = "*"
 geopy = "*"
 dnspython = "*"
 algorithmia = "*"
-asana = "*"
+asana = "<=3.2.1"  # BC in Higher version
```
Comment (JudahSchwartz) [inline docker/py3-tools/pyproject.toml:18]:
```suggestion
asana = "<=3.2.1"  # BC in Higher version, community supported integration
```

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**